src/components/VmsPageHeader/style.css

```diff
@@ -0,0 +1,2 @@
+.user-name {
```

Comment:
@jniederm , please do not hesitate and clean up similar things.

Anyway, since these changes are hard to track/review, **please be thorough in mentioning** them in commit or PR comments.